BlazorWeather.Maui/Platforms/Windows/Package.appxmanifest

```diff
@@ -7,12 +7,12 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="3AEFAA61-7D58-4517-BC3D-1E5387CC1917"
+    Name="9B2EF06F-B348-4CE9-8165-554E3A82D16B"
     Publisher="CN=User Name"
     Version="1.0.0.0" />
 
   <Properties>
-    <DisplayName>Blazor Weather</DisplayName>
+    <DisplayName>BlazorWeather.Maui</DisplayName>
```

Comment:
These app names need to be manually updated still. We are still working on making this be editable from the csproj.